### PR TITLE
Add dashboard component

### PR DESCRIPTION
**Pull request by erlikh**

### CODE REVIEW

File: src/actions/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return { type: TODOS_REQUEST };
+}
+
+export function fetchTodos () {
+  const todosUrl = 'http://syncidsapi.azurewebsites.net/api/todos?firmName=syncidscom&q=syncids.com&showAll=false';
+  return (dispatch) => {
+    dispatch(requestTodos());
+    return axios
+      .get(todosUrl)
+      .then(json => json.data)
```
Comment (ntgn81) [inline src/actions/dashboard.js]: Unnecessary step, or did you have a reason for this?

Comment (erlikh) [inline src/actions/dashboard.js]: Just though it is better for reducer not to know about shape of response we get from backend

Comment (ntgn81) [inline src/actions/dashboard.js]: Ya, I meant to just `dispatch(receiveTodos(json.data))`

**Comment by ntgn81** [comment]
:+1: Merging

**Pull request merged**